src/main/java/com/softserve/teamproject/entity/Country.java

```diff
@@ -56,6 +57,27 @@ public void setLocations(Set<Location> locations) {
   }
 
   @Override
+  public boolean equals(Object otherObject) {
+    if (this == otherObject) {
+      return true;
+    }
+    if (otherObject == null) {
+      return false;
+    }
+    if (getClass() != otherObject.getClass()) {
+      return false;
+    }
+    Country other = (Country) otherObject;
+    return Objects.equals(id, other.id) && Objects.equals(name, other.name);
+//        && Objects.equals(locations, other.locations);
```

Comment:
I'm not sure would we need this check in future? If not, could you please delete it.